Metadata/Driver/AnnotationDriver.php

```diff
@@ -72,6 +72,10 @@ public function loadMetadataForClass(\ReflectionClass $class)
 
         foreach ($this->reader->getClassAnnotations($class) as $annot) {
             if ($annot instanceof Service) {
+                if (!isset($annot->decorationInnerName) && isset($annot->decoration_inner_name)) {
```

Comment:
@Ener-Getick What is your opinion on this? If you still think this message should be removed, I will... but I do have the feeling they are two different messages.